text/0000-edition-2021.md

```diff
@@ -0,0 +1,226 @@
+- Feature Name: edition_2021
+- Start Date: 2021-02-19
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC describes the plan for the 2021 Edition. It supercedes [RFC 2052]. The proposed 2021 Edition is intentionally more limited than the 2018 Edition. Rather than representing a major marketing push, the 2021 Edition represents a chance for us to introduce changes that would otherwise be backwards incompatible. It is meant to be marketed in many ways as something closer to "just another release".
+
+Key points include:
+
+* Editions are used to introduce changes into the language that would otherwise have the potential to break existing code, such as the introduction of a new keyword.
+* Editions are never allowed to split the ecosystem. We only permit changes that still allow crates in different editions to interoperate.
+* Editions are named after the year in which they occur (e.g., Rust 2015, Rust 2018, Rust 2021).
+* When we release a new edition, we also release tooling to automate the migration of crates. Some manual work may be required but that should be the rare case.
```

Comment:
Nitpick, but to me the word "rare" seems to suggest that you think the automated migration should almost never fail. Did you have that strong of an assertion in mind? If what you had in mind was more along the lines of "the automated migration should probably always succeed", then a softer word like "uncommon" might set expectations better.